core/kernel/persistence/smoothsql/search/driver/TaoSearchDriver.php

```diff
@@ -106,6 +106,7 @@ public function like()
         $like = [
             'mysql'      => 'LIKE',
             'postgresql' => 'ILIKE',
+            'gcp-spanner' => 'LIKE'
```

Comment:
can you explain the context of this fix (spanner wise), are we not using the mysql one while running spanner?